<ui version="4.0" >
 <class>formWallpaper</class>
 <widget class="QWidget" name="formWallpaper" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item rowspan="5" row="0" column="0" >
      <widget class="QListWidget" name="wallpaperList" >
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QToolButton" name="buttonAddNewWallpaper" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Add new wallpaper</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="arrowType" >
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QToolButton" name="buttonSetFillColor" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Fill color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QLabel" name="labelWallpaperPreview" >
       <property name="minimumSize" >
        <size>
         <width>160</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>160</width>
         <height>120</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="midLineWidth" >
        <number>1</number>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <spacer name="verticalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" >
      <widget class="QPushButton" name="buttonRevert" >
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <widget class="QPushButton" name="buttonApply" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
